src/core/ext/transport/cronet/transport/cronet_transport.c

```diff
@@ -151,6 +151,11 @@ struct op_state {
   bool state_callback_received[OP_NUM_OPS];
   bool fail_state;
   bool flush_read;
+#ifdef GRPC_CRONET_WITH_PACKET_COALESCING
```

Comment:
@makdharma is current framework allow any configuration information (e.g. enable_packet_coalescing) into cronet_transport? Using initial metadata seems to be a way to me but I am wondering if there is any dedicated approach.